crypto/threads_pthread.c

```diff
@@ -10,6 +10,10 @@
 #include <openssl/crypto.h>
 #include "internal/cryptlib.h"
 
+# if defined(__sun)
+#include <atomic.h>
+# endif
```

Comment:
Code style: Indentation is off, should look like this:

``` C
#if defined(__sun)
# include <atomic.h>
#endif
```

Reply:
fixed